apps/api/src/app/messages/messages.controller.ts

```diff
@@ -71,4 +78,26 @@ export class MessagesController {
       })
     );
   }
+
+  @Delete('')
```

Comment:
```suggestion
  @Delete('/transaction/:transactionId')
```

Not sure regarding the current implementation, we either need to have a general remove endpoint with an optional filter, or a more specific endpoint for this.

@LetItRock wdyt here?